### PR TITLE
Formatting docs

### PR DESCRIPTION
**Pull request by dreamy-dev**

### CODE REVIEW

File: docs/static/style-guide/formats.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+#### `sszvis.formatText(x)`
+
+Any given data is returned as a string and will be formatted as Text.
+
+```table
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
```
Comment (dreamy-dev) [inline docs/static/style-guide/formats.md]: Used this to show that the returned format is a string. Not sure if I formatted this correctly.

File: docs/static/style-guide/formats.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    Result: "16 000"
+  - Function:
+    Example: sszvis.formatNumber(5000)
+    Result: "5000"
+```
+
+#### `sszvis.formatPreciseNumber(p, d)`
+
+`p` = The desired precision
+`d` = The number to be formatted
```
Comment (dreamy-dev) [inline docs/static/style-guide/formats.md]: Added these for a more detailed explanation of the function

File: docs/static/style-guide/formats.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
+  - Function:
+    Example: sszvis.formatText("Jahr")
+    Result: "'Jahr'"
+```
+
+#### `sszvis.formatNumber(x)`
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]: Please also add some examples that show how numbers will be shortened if they have insignificant decimals. See `test/format.test.js` for more examples.

File: docs/static/style-guide/formats.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
+> This library provides various functions for formatting the data in the preferred style.
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]:
```suggestion
> This library provides functions for formatting data following the SSZVIS style guide.
```

File: docs/static/style-guide/formats.md (hunk 2 above)
Comment (grossbart) [reply docs/static/style-guide/formats.md]: I like these!

File: docs/static/style-guide/formats.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+```
+
+#### `sszvis.formatPreciseNumber(p, d)`
+
+`p` = The desired precision
+`d` = The number to be formatted
+
+A number will be formatted to a particular precision. It is a function that accepts numbers and returns formatted values. This function is "curried", meaning that it is a function with multiple arguments, but when you call it with less than the full number of arguments, it returns a function that takes less arguments and has the arguments you did provide "pre-filled" as parameters.
+
+Note that preciseNumber(2, 14.1234) is equivalent to preciseNumber(2)(14.1234)
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]:
```suggestion
Note that `preciseNumber(2, 14.1234)` is equivalent to `preciseNumber(2)(14.1234)`.
```

File: docs/static/style-guide/formats.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+```table
+rows:
+  - Function: formatPreciseNumber
+    Example: sszvis.formatPreciseNumber(2, 14.1234)
+    Result: 14.12
+  - Function:
+    Example: sszvis.formatPreciseNumber(3, 14.1234)
+    Result: 14.123
+  - Function:
+    Example: sszvis.formatPreciseNumber(3)(14.1234)
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]: This is duplicated

File: docs/static/style-guide/formats.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+    Result: 14.123
+  - Function:
+    Example: sszvis.formatPreciseNumber(0, 14.1234)
+    Result: 14
+```
+
+#### `sszvis.formatPercent(x)`
+
+A number will be transformed into a percent format. Formats percentages on the range 0 - 100.
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]:
```suggestion
A number will be transformed into a percent format. Formats percentages on the range 0–100.
```

File: docs/static/style-guide/formats.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+    Example: sszvis.formatPercent(7)
+    Result: "7 %"
+  - Function:
+    Example: sszvis.formatPercent(0.245875)
+    Result: "0.25 %"
+```
+
+#### `sszvis.formatFractionPercent(x)`
+
+A number will be transformed into a percent format. Format percentages on the range 0 - 1.
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]:
```suggestion
A number will be transformed into a percent format. Format percentages on the range 0–1.
```

File: docs/static/style-guide/formats.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+
+A number will be formatted as an age. Means that any decimal number will be rounded to a full number.
+
+```table
+rows:
+  - Function: formatAge
+    Example: sszvis.formatAge(55.54359375)
+    Result: "56"
+  - Function:
+    Example: sszvis.formatAge(555435)
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]: This is not a very realistic age 😅

File: docs/static/style-guide/formats.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+#### `sszvis.formatAxisTimeFormat(x)`
+
+A multi time formatter used by the axis class.
+
+```table
+rows:
+  - Function: formatAxisTimeFormat
+    Example: sszvis.formatAxisTimeFormat(new Date())
+    Result: ".345"
```
Comment (grossbart) [inline docs/static/style-guide/formats.md]: This is an important function, but this example doesn't show what it does very well. Maybe we should not show an example and just say that it is used by d3 under the hood. (See https://github.com/d3/d3-time-format)

File: docs/static/style-guide/formats.md (hunk 6 above)
Comment (dreamy-dev) [reply docs/static/style-guide/formats.md]: Yes, but on purpose to demonstrate both ways to write this function.

File: docs/static/style-guide/formats.md (hunk 10 above)
Comment (dreamy-dev) [reply docs/static/style-guide/formats.md]: Good input, but how do we explain when it can be used?

**Comment by dreamy-dev** [comment]
PR formatting docs

**Pull request merged**